docs/api/data.rst

```diff
@@ -167,6 +167,21 @@ All of the other methods in the :class:`Genotypes` class are inherited, but the
 
 .. _api-data-genotypesplink:
```

Comment:
```suggestion
.. _api-data-genotypestr:
```

I think this should fix the readthedocs build error. It's complaining because there are two sections labeled with the "api-data-genotypesplink" ID